config_template.cfg

```diff
@@ -1143,19 +1148,19 @@ MULTIPOINT_WEIGHT= (0.33333, 0.33333, 0.33333)
 % Optimization objective function with scaling factor, separated by semicolons.
 % To include quadratic penalty function: use OPT_CONSTRAINT option syntax within the OPT_OBJECTIVE list.
 % ex= Objective * Scale
-OPT_OBJECTIVE= DRAG * 0.001
+OPT_OBJECTIVE= DRAG
```

Comment:
Why is that removed ?

Reply:
If you use OPT_RELAX_FACTOR and OPT_GRADIENT_FACTOR instead of the objective function scale factor you obtain much better results.

Reply:
OPT_RELAX_FACTOR and OPT_GRADIENT_FACTOR are the correct variables to use in optimization problems for the new release